src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -33,41 +33,20 @@ use crate::prost_key_range::KeyRangeExt;
 use crate::{can_concat, CompactionGroupId, HummockSstableId, HummockSstableObjectId};
 
 pub struct GroupDeltasSummary {
```

Comment:
We can store an array of `IntralLevelDelta` here.

Reply:
yes, fix it